pkg/tfbridge/provider.go

```diff
@@ -669,7 +669,25 @@ func DiffConfig(
 		schemaMap:   config,
 		schemaInfos: configInfos,
 	}
-	return differ.DiffConfig
+
+	diff := func(
+		urn resource.URN, oldInputs, oldOutputs, newInputs resource.PropertyMap,
+		allowUnknowns bool, ignoreChanges []string,
+	) (plugin.DiffResult, error) {
+		return differ.DiffConfig(
+			context.TODO(),
```

Comment:
I've done this to avoid changing the signature of this function to accept a context, because I assume that will break other things, but happy to do that, or change this to e.g. `context.Background()`.

Reply:
I'd guess this is ok to change the signature since it's only used in pf and that already has a context.

I don't imagine this being used externally.